cypress/e2e/process.cy.js

```diff
@@ -156,7 +157,7 @@ describe('Processes menu', {tags: tags}, ()=> {
             tenureReviewDocsPage.buttonConfirm().click();
             cy.contains('Thank you for your confirmation');
             cy.contains("This case is now closed and we have recorded this on the system - that you have sent an outcome letter to the resident. The outcome can be viewed in the activity history");
-            cy.contains("a", "Return to home page").should("have.attr", "href");
+            cy.url().should("include", "manage-my-home")

```

Comment:
Won't every single MMH page pass this criteria?
`https://manage-my-home-staging.hackney.gov.uk/`

Reply:
Yep - not my change. Weird git rebase things